File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            int pivote = nums[i]; // Declaración y asignación
            for (int j = i + 1; j < nums.length; j++) { // Evita comparar el mismo elemento
                if (pivote == nums[j]) {
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        i = 0
        output = []
        while i < len(nums):
            mult = 1
            j = 0
            while j < len(nums):
                if(i != j):
                    mult *= nums[j]
                j += 1
            output.append(mult)
            i += 1
        return output